plugins/inputs/gtfs/gfts_test.go

```diff
@@ -0,0 +1,191 @@
+package gtfs_test
```

Comment:
No this should be
```suggestion
package gtfs
```

Reply:
Testing a package as an external consumer is an idiomatic pattern in Go. This ensures the subject package is exercised as a consumer would exercise it, and avoids common pitfalls in unit testing where the test becomes less meaningful because it has too much knowledge of and interaction with package internals. These tests do not need internal access to the package and are therefore not given it.

Further evidence that this is idiomatic and intended is the fact that a directly may only contain one package in Go, except for this specific case, where `package foo` is accompanied by a black-box `package foo_test`.

Reply:
Well almost all plugins in telegraf put the test into the same package with the main code, probably because there is no external customer...